python/ground/common/model/core/structure.py

```diff
@@ -0,0 +1,37 @@
+from ..version.item import Item
```

Comment:
Is there no better way to do this? Okay if not, but seems clunky.

Reply:
Updated these imports to be absolute